core/encoding/cli.go

```diff
@@ -31,7 +33,7 @@ func CLIFlags(envPrefix string) []cli.Flag {
 		cli.StringFlag{
```

Comment:
Now that G2 path is no longer required. I think it should be validated in `NewKzgEncoderGroup` (if it's loading g2 points, this file is required. Otherwise, `G2PowerOf2Path` is required).

Reply:
make sense